File: SofiForce.Web/ClientApp/src/app/core/Models/SearchModels/OperationRequestSearchModel.ts
import { BaseSearchModel } from "./baseSearchModel";

export interface OperationRequestSearchModel extends BaseSearchModel {
    operationCode: string;
    operationTypeId: number | null;
    representativeId: number | null;
    representativeCode: string | null;
    governerateId:number |null;
    operationDate: Date | null;
    startDate: Date | null;
    isClosed: number | null;
}

File: SofiForce.Models/Models/SearchModels/OperationRequestSearchModel.cs
using System;
using System.Collections.Generic;

namespace Models
{
    public class OperationRequestSearchModel : BaseSearchModel
    {
		public string OperationCode { get; set; }

	
		public Int32? GovernerateId { get; set; }
		public Int32? OperationTypeId { get; set; }
		public Int32? RepresentativeId { get; set; }
		public DateTime? OperationDate { get; set; }
		public DateTime? StartDate { get; set; }

		public Int32? IsClosed { get; set; }


	}
}